bevy_lint/action.yml

```diff
@@ -9,6 +9,28 @@ description: |
 
   Note that this currently will override the default Rustup toolchain to nightly. You can fix this
   by calling `rustup default` after this action is run.
+inputs:
+  cache:
+    description: |
+      If set to `true`, this action will cache the built `bevy_lint` executable and reuse it on
+      subsequent runs, which can speed up the runtime. This defaults to `false`.
+    required: false
+    default: 'false'
+  save-cache-if:
+    description: |
+      When caching is enabled, this configures whether a freshly built `bevy_lint` will be saved as
+      a new cache. When caching is disabled, this does nothing. This defaults to `true`.
+    required: false
+    default: 'true'
+outputs:
+  cache-hit:
+    description: |
+      A string indicating whether an existing cached `bevy_lint` was used. This will be 'true' if a
+      cache was used and '' (an empty string) if a cache was not. If caching is disabled, this will
+      always be an empty string.
+    # Technically `cache-hit` could be 'false' if a fallback key was used instead of the primary
+    # key, but we don't use fallback keys, so that shouldn't happen.
+    value: ${{ steps.cache.outputs.cache-hit || steps.restore-cache.outputs.cache-hit }}
```

Comment:
While I would much rather this be `'true'` or `'false'` for if the cache was hit, `actions/cache` uses `'true'` and `''`. I'd rather follow precedence here than make things confusing and different.